src/librustc_mir/lib.rs

```diff
@@ -24,16 +24,18 @@ Rust MIR: a lowered representation of Rust. Also: an experiment!
 #![feature(collection_placement)]
 #![feature(nonzero)]
 
+#[allow(unused_extern_crates)]
```

Comment:
Can you add a comment here pointing to the `Cargo.toml` of rustc_cratesio_shim?

Reply:
Done.